src/main/java/school/lemon/changerequest/java/introduction/hw2/Task1.java

```diff
@@ -1,14 +1,12 @@
 package school.lemon.changerequest.java.introduction.hw2;
 
-import java.io.PrintStream;
-
 public class Task1 {
     public static String makeTags(String tag, String text) {
 
-        return (new StringBuilder("<").append(tag).append(">").append(text).append("</").append(tag).append(">")).toString();
+        return String.format("<" + tag + ">" + text + "</" + tag + ">");
```

Comment:
Please, read about String.format usage and prepare `pattern` (first argument in String.format) and `parameters` (all other arguments).
You can check following link for examples: [String.format() examples](https://examples.javacodegeeks.com/core-java/lang/string/java-string-format-example/)